src/panels/config/zha/zha-device-card.ts

```diff
@@ -81,6 +81,12 @@ class ZHADeviceCard extends LitElement {
     super.connectedCallback();
     this._unsubAreas = subscribeAreaRegistry(this.hass.connection, (areas) => {
```

Comment:
You should btw look into the `subscribeMixin` https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/mixins/subscribe-mixin.ts it handles the right moment to subscribe and unsubscribe.